src/microbe_stage/systems/MicrobeAISystem.cs

```diff
@@ -755,11 +779,6 @@ ref otherMicrobeInfo.Entity.Get<SpeciesMember>(), speciesOpportunism, speciesFoc
                 // If the predator is right on top of us there's a chance to try and swing with a pilus
                 ai.MoveWithRandomTurn(2.5f, 3.0f, position.Position, ref control, speciesActivity, random);
             }
-
-            // Sprint until full strain
```

Comment:
This seems to revert the AI running PR changes?

Reply:
Is it fine to re-add in new commit? I probably accidentally did it while merging and thus has other stuff in its commit.

Reply:
It should be totally fine. This PR is likely going to be squash merged so this hiccup won't even show up on the master branch history.